ecs-cli/modules/cli/cluster/cluster_app.go

```diff
@@ -305,17 +358,15 @@ func deleteCluster(context *cli.Context, rdwr config.ReadWriter, ecsClient ecscl
 	// Validate that a cfn stack exists for the cluster
 	cfnClient.Initialize(cliParams)
 	stackName := cliParams.CFNStackName
-	if err := cfnClient.ValidateStackExists(stackName); err != nil {
-		return fmt.Errorf("CloudFormation stack not found for cluster '%s'", cliParams.Cluster)
-	}
 
-	// Delete cfn stack
-	if err := cfnClient.DeleteStack(stackName); err != nil {
-		return err
-	}
-	logrus.Info("Waiting for your cluster resources to be deleted...")
-	if err := cfnClient.WaitUntilDeleteComplete(stackName); err != nil {
-		return err
+	if err := cfnClient.ValidateStackExists(stackName); err != nil {
+		if err := confirmDeleteWithoutStack(cliParams); err != nil {
```

Comment:
We should check if the force flag was given or not here, and only print this message if force is not specified. Having commands that require responding to a prompt is problematic to anyone who wants to use the ECS CLI in an automated script. Using `--force` to ignore all prompts allows a user to use the CLI in a script.